pkg/otel/config.go

```diff
@@ -320,11 +319,10 @@ func buildApertureSelfScrapeConfig(name string, cfg *otelParams) map[string]any
 
 func buildKubernetesNodesScrapeConfig(cfg *otelParams) map[string]any {
 	return map[string]any{
-		"job_name":        "kubernetes-nodes",
-		"scheme":          "https",
-		"scrape_interval": "2s",
-		"scrape_timeout":  "1500ms",
-		"metrics_path":    "/metrics/cadvisor",
+		"job_name":       "kubernetes-nodes",
+		"scheme":         "https",
+		"scrape_timeout": "1500ms",
+		"metrics_path":   "/metrics/cadvisor",
```

Comment:
How often are cadvisor metrics being scraped?

Reply:
Every 1 second for control loop and every 10 seconds for FN plugin data.